docs/api/remix.md

```diff
@@ -2032,27 +2032,36 @@ Remix comes with several pre-built session storage options for common scenarios,
 
 This is an example of a cookie session storage:
 
-```js filename=app/sessions.js
-// app/sessions.js
+```ts filename=app/sessions.ts
 import { createCookieSessionStorage } from "@remix-run/node"; // or "@remix-run/cloudflare"
 
-const { getSession, commitSession, destroySession } =
-  createCookieSessionStorage({
-    // a Cookie from `createCookie` or the CookieOptions to create one
-    cookie: {
-      name: "__session",
+export interface SessionData {
+  userId: string;
+}
```

Comment:
It's not needed to export these interfaces.
Let's also make it a `type` instead, so it's clear that we're using it locally & not doing interface magic ([declaration merging](https://www.typescriptlang.org/docs/handbook/declaration-merging.html)) here.

```suggestion
type SessionData = {
  userId: string;
};
```